docs/data/tree-view/simple-tree-view/customization/BorderedTreeView.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import IndeterminateCheckBoxRoundedIcon from '@mui/icons-material/IndeterminateCheckBoxRounded';
+import DisabledByDefaultRoundedIcon from '@mui/icons-material/DisabledByDefaultRounded';
+import AddBoxRoundedIcon from '@mui/icons-material/AddBoxRounded';
+import { styled, alpha } from '@mui/material/styles';
+import { SimpleTreeView } from '@mui/x-tree-view/SimpleTreeView';
+import { TreeItem, treeItemClasses } from '@mui/x-tree-view/TreeItem';
+
+const CustomTreeItem = styled(TreeItem)(({ theme }) => ({
+  [`& .${treeItemClasses.content}`]: {
+    padding: theme.spacing(0.5, 1),
+    margin: theme.spacing(0.2, 0),
+    [`& .${treeItemClasses.label}`]: {
+      fontFamily: 'General Sans',
+      fontWeight: 500,
+      fontSize: '0.8rem',
+    },
```

Comment:
```suggestion

```

I think we can remove the custom typography styles here! Just to focus _specifically_ on the connection border customization.